src/Lucene.Net/Index/IndexWriter.cs

```diff
@@ -2669,11 +2687,17 @@ internal System.String NewSegmentName()
 		/// </summary>
 		/// <throws>  CorruptIndexException if the index is corrupt </throws>
 		/// <throws>  IOException if there is a low-level IO error </throws>
-		/// <seealso cref="Index.LogMergePolicy.FindMergesForOptimize">
+		/// <seealso cref="Net.Index.LogMergePolicy.FindMergesForOptimize">
 		/// </seealso>
-		public virtual void  Optimize(IState state)
+		public virtual void  Optimize(IState state, CancellationToken token)
```

Comment:
We can assign default value to `token` so you won't need to add `CancellationToken.None` in all places

Reply:
`CancellationToken token = default(CancellationToken)`

Reply:
done